temporal-sdk/src/main/java/io/temporal/internal/worker/LocalActivityWorker.java

```diff
@@ -106,6 +106,7 @@ private void submitAttempt(
               TimeUnit.MILLISECONDS);
     }
 
+    executionContext.newAttempt();
```

Comment:
It should be inside AttemptTaskHandlerImpl#AttemptTaskHandlerImpl#handle right after `attemptTask.markAsTakenFromQueue();`

Reply:
Done